pkg/targetgroupbinding/resource_manager.go

```diff
@@ -415,6 +414,24 @@ func (m *defaultResourceManager) updateTargetHealthPodConditionForPod(ctx contex
 	return needFurtherProbe, nil
 }
 
+func (m *defaultResourceManager) calculateReadinessGateTransition(pod k8s.PodInfo, targetHealthCondType corev1.PodConditionType, targetHealth *elbv2types.TargetHealth) (string, string, corev1.ConditionStatus, bool) {
```

Comment:
Lets just compute the `targetHealthCondStatus` and if `anyFurtherProbeisNeeded` in this as Reason and message are not important for updating the tgb checkpoint. This makes it clear in reading.

Reply:
done :)